<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="formulaTableWidgetList">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>411</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Ingredient Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBoxChemical">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Chemical Properties</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>261</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelChemical">
        <property name="text">
         <string>Chemical</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxChemical"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>261</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelAmount">
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxChemicalAmount"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxOperatingConditions">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Operating Conditions</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="tempOfIngredientsLabel">
         <property name="text">
          <string>Temp of Ingredients (K)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="tempOfIngredientsDoubleSpinBox"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="tempOfIngredientsLabel_2">
         <property name="text">
          <string>Chamber Pressure (PSI)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="chamberPressureDoubleSpinBox"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="tempOfIngredientsLabel_3">
         <property name="text">
          <string>Exhaust Pressure (PSI)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="exhaustPressureDoubleSpinBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxFormulaResults">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>731</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Formula Results</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelChamberCPCV">
         <property name="text">
          <string>Chamber CP/CV:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelChamberCPCVResult">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="labelIsp">
         <property name="text">
          <string>Isp*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelIspResult">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="labelChamberTemp">
         <property name="text">
          <string>Chamber Temp:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelChamberTempResult">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="labelC">
         <property name="text">
          <string>C*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCResult">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="labelMolecularWt">
         <property name="text">
          <string>Molecular Wt:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMolecularWtResult">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="labelDensity">
         <property name="text">
          <string>Density</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDensityResult">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QPushButton" name="removeChemicalPushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editChemicalPushButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addChemicalPushButton">
       <property name="text">
        <string>Add </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExcel_Mixing_Sheet"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionPepcoded_File"/>
   </widget>
   <widget class="QMenu" name="menuCalculate">
    <property name="title">
     <string>Calculate</string>
    </property>
    <addaction name="Calculate"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutOpenPep"/>
    <addaction name="actiontest12"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCalculate"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="Calculate">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAboutOpenPep">
   <property name="text">
    <string>About OpenPep</string>
   </property>
  </action>
  <action name="actiontest12">
   <property name="text">
    <string>test12</string>
   </property>
  </action>
  <action name="actionExcel_Mixing_Sheet">
   <property name="text">
    <string>Excel Mixing Sheet</string>
   </property>
  </action>
  <action name="actionPepcoded_File">
   <property name="text">
    <string>Pepcoded File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
